pkgs/applications/networking/cluster/terraform-providers/providers.json

```
@@ -1141,6 +1141,15 @@
     "vendorSha256": null,
     "version": "1.31.0"
   },
+  "utils": {
+    "owner": "cloudposse",
+    "provider-source-address": "registry.terraform.io/cloudposse/terraform-provider-utils",
```

`cloudposse/terraform-provider-utils` doesn't exist on the registry.
```suggestion
    "provider-source-address": "registry.terraform.io/cloudposse/utils",
```